includes/amp-helper-functions.php

```diff
@@ -85,7 +85,7 @@ function amp_get_permalink( $post_id ) {
 	if ( current_theme_supports( 'amp' ) ) {
 		$permalink = get_permalink( $post_id );
 		if ( ! amp_is_canonical() ) {
-			$permalink = add_query_arg( 'amp', '', $permalink );
+			$permalink = add_query_arg( 'amp', '1', $permalink );
```

Comment:
If we go ahead with this PR, then what we should do is take the opportunity to factor out the redundant formation of the AMP URLs with a new common function. There is presently `amp_get_permalink()` but that is specifically for getting URLs for posts. The `amp_get_url()` function would be used inside of `amp_get_permalink()`.

There could be an `amp_get_url()` function that looks like this:

```php
/**
 * Get the AMP version of a URL.
 *
 * This simply appends the required query parameter to the URL. It does not confirm
 * whether or not AMP is allowed on the given URL. If AMP is not supported, then a redirect
 * to the non-AMP version will ensue.
 *
 * @see amp_get_permalink()
 * @return string AMP URL.
 */
function amp_get_url( $url ) {
    return add_query_arg( amp_get_slug(), '1', $url );
}
```

Reply:
@westonruter  I'll work on it...

Reply:
Hello @westonruter! Getting back to this now...